daliuge-engine/dlg/drop.py

```diff
@@ -387,7 +387,9 @@ def getmembers(object, predicate=None):
             elif isinstance(obj, dlg_dict_param):
                 value = kwargs.get(attr_name, obj.default_value)
                 if isinstance(value, str):
-                    value = ast.literal_eval(value)
+                    value = {}
```

Comment:
This looks like it will always set value = {}
- Enters branch since value is a str
- Sets value to {}
- calls ast.literal_eval on value (which is {})